html/images/os/coriant.svg

```diff
@@ -0,0 +1,63 @@
+     id="defs106" /><path
+     style="fill:#00467f"
+     id="path79"
+     d="M 79,30.334 95,18.667 C 95,18.667 84.333,0 56.333,0 28.333,0 0,21.667 0,57.667 0,93.667 29,113 58.667,113 c 29.667,0 39,-19.667 39,-19.667 L 81,81.001 c 0,0 -8.666,13 -23.333,13 -14.667,0 -35.333,-10 -35.333,-37.5 0,-27.5 21,-36.5 34,-36.5 13,0 22.666,10.333 22.666,10.333 z" /><g
+     transform="translate(-22,-346.666)"
+     id="g83"><path
+       style="fill:#00467f"
+       id="path81"
+       d="m 161.667,381.667 c 0,0 -40.333,1.333 -40.333,39.333 0,38 38.666,38.667 38.666,38.667 0,0 41.667,-2 41.667,-39 0,-37 -40,-39 -40,-39 z M 161.5,440.43 c -11.414,0 -20.667,-9.254 -20.667,-20.667 0,-11.414 9.253,-20.667 20.667,-20.667 11.414,0 20.667,9.253 20.667,20.667 0,11.413 -9.253,20.667 -20.667,20.667 z" /></g><path
+     style="fill:#00467f"
+     id="path85"
+     d="m 205.333,110.668 h -20 V 37.334 h 20 v 10 c 0,0 9.333,-17.333 28.333,-10 0.333,7.333 0,18 0,18 0,0 -22.667,-7.667 -28,13.333 0.001,8.334 -0.333,42.001 -0.333,42.001 z" /><path
+     style="fill:#00467f"
+     id="path87"
+     d="M 242.333,37.667" /><rect
+     style="fill:#00467f"
+     id="rect89"
+     height="73"
+     width="20.667"
+     y="37.667007"
+     x="242.33301" /><rect
+     style="fill:#00467f"
+     id="rect91"
+     height="20"
+     width="20.667"
+     y="10.000992"
+     x="242.33301" /><path
+     style="fill:#00467f"
+     id="path93"
+     d="m 364.667,110.668 h -19.333 v -73 h 19.333 v 8.333 c 0,0 4.667,-11 20,-11 15.333,0 27.667,11 27.667,23.333 0,12.333 0,52.333 0,52.333 h -20 V 67 c 0,0 0,-14 -13.5,-14 -8.5,0 -13.834,10 -13.834,15.333 0,5.333 -0.333,42.335 -0.333,42.335 z" /><path
+     style="fill:#00467f"
+     id="path95"
+     d="M 431.334,55.001 H 417 V 37.668 h 14.334 V 17.001 h 20 v 20.667 h 12.333 L 472,55.001 h -20.333 v 29.667 5 c 0,0 1.001,7 9.667,5 8.666,-2 8.666,-2 8.666,-2 l 2,16 c 0,0 -8.666,8 -29.333,2 C 430.334,105.668 431,92.001 431,82.001 c 0,-10 0.334,-27 0.334,-27 z" /><circle
+     style="fill:#00467f"
+     id="circle97"
+     r="13.167"
+     cy="101.834"
+     cx="491.83301" /><path
+     style="fill:#008c99"
+     id="path99"
+     d="m 487.334,64.95 -8.667,-18.283 c 0,0 59.999,-14.333 70.333,55.167 0,59.5 -80,79.833 -109.666,21.166 14,3.667 25.333,1.667 25.333,1.667 0,0 8.334,12.999 27.167,12.333 18.833,-0.666 35.833,-11.999 35.833,-33.666 0,-21.667 -15.999,-37.768 -40.333,-38.384 z" /><path
+     style="fill:#00467f"
+     id="path101"
+     d="M 335.667,58.334 C 334.334,45.667 324,35.001 304,35.001 c -20,0 -31.333,13 -31.333,13 l 10.667,10.333 c 0,0 7.666,-7.333 21.333,-7.333 13.667,0 14,13.333 14,13.333 0,0 -1.667,0 -16.204,0 -14.537,0 -33.796,8 -33.796,23.667 0,15.667 12.333,24 26,25 13.667,1 23.333,-9.667 23.333,-9.667 v 7.334 h 17.667 c 0,0 1.333,-39.667 0,-52.334 z m -19,28.333 c 0,0 -2.667,10.667 -14.204,10.667 C 294,97.334 289,94 289,88.667 c 0,-5.333 9.334,-10.666 17.667,-10.333 8.333,0.333 10,0 10,0 z" /></svg>
```

Comment:
Can you run this through something like svgo to compress it down a bit (https://github.com/svg/svgo)